packages/forms-web-app/tests/unit/controllers/appellant-submission/upload-application.test.js

```diff
@@ -119,7 +119,7 @@ describe('controllers/appellant-submission/upload-application', () => {
       expect(res.redirect).toHaveBeenCalledWith(fakeNextUrl);
     });
 
-    it('should redirect to `/appellant-submission/supporting-documents` if valid', async () => {
+    it('should redirect to `/appeal-householder-decision/supporting-documents` if valid', async () => {
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks